src/Ubiquity/contents/validation/validators/strings/HashValidator.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Ubiquity\contents\validation\validators\strings;
+
+/**
+ * Validates an email address
+ * Usage @validator("ahsentekdemir.sh@gmail.com")
```

Comment:
The usage would be more like:
```php
 * Usage @validator("regex","sha-1")
```

Reply:
I think you really need to specify the possible uses of this validator, I do not see well